src/config.py

```diff
@@ -7,65 +7,47 @@
 class Config:
     """Reads Platform.sh configuration from environment variables.
 
-    https://docs.platform.sh/development/variables.html
+    See: https://docs.platform.sh/development/variables.html
 
     The following are 'magic' properties that may exist on a Config object.
     Before accessing a property, check its existence with hasattr(config, variableName).
     Attempting to access a nonexistent variable will throw an exception.
 
-    Attributes
-    ----------
-
-    directVariables
-        Local index of the variables that can be accessed as direct properties (build and runtime).
-        The key is the property that will be read.
-        The value is the environment variables, minus prefix, that contains the value to look up.
-    directVariablesRuntime
-        Local index of the variables that can be accessed as direct properties (runtime only).
-        The key is the property that will be read.
-        The value is the environment variables, minus prefix, that contains the value to look up.
-    environmentVariables
-        A local copy of all environment variables as of when the object was initialized
-    envPrefix
-        The vendor prefix for all environment variables we care about.
-    routesDef
-        The routes definition array. Only available at runtime.
-    relationshipsDef
-        The relationships definition array. Only available at runtime.
-    variablesDef
-        The variables definition array. Available in both build and runtime, although possibly
-        with different values.
-    applicationDef
-        The application definition array. This is, approximately, the .platform.app.yaml file
-        in nested array form.
-
-    These properties are available at build time and run time:
-    ----------------------------------------------------------
-
-    project
-        The project ID.
-    applicationName
-        The name of the application, as defined in its configuration.
-    treeId
-        An ID identifying the application tree before it was built: a unique hash
-        is generated based on the contents of the application's files in the repository.
-    appDir
-        The absolute path to the application.
-    entropy
-        A random string generated for each project, useful for generating hash keys.
-
-    These properties are only available at runtime:
-    -----------------------------------------------
-
-    branch
-        The Git branch name.
-    environment
-        The environment ID (usually the Git branch plus a hash).
-    documentRoot
-        The absolute path to the web root of the application.
-    smtpHost
-        The hostname of the Platform.sh default SMTP server (an empty string
-        if emails are disabled on the environment.
+    Attributes:
+        directVariables (dict): Local index of the variables that can be accessed as direct properties (build and
```

Comment:
Multiple lines are acceptable in Google-style docstrings. I would do that here. Having the attribute name and description on separate lines is way more readable